README.md

```diff
@@ -1,13 +1,47 @@
+Individuals Charges API
+========================
+This API allows software packages to show and provide a taxpayers’ financial data for their charges.
 
-# individuals-charges-api
+For pension charges a developer can:
 
-# To run locally
+* retrieve pension charges
+* create and amend pension charges
+* delete pension charges
 
-`sbt "run 9765"`
-or
-`sbt run`
+## Requirements
+- Scala 2.12.x
+- Java 8
+- sbt 1.3.7
+- [Service Manager](https://github.com/hmrc/service-manager)
 
-### License
+## Development Setup
+
+Run from the console using: `sbt run` (starts on port 9765 by default)
+
+Start the service manager profile: `sm --start MTDFB_ALL`
+
```

Comment:
For the service manager profile I believe if you look in the service-manager-config repo you should be able to find a profile more specific than all the APIs